src/modules/gameScore.js

```diff
@@ -0,0 +1,51 @@
+class GameScore {
+  constructor(user, score) {
+    this.user = user;
+    this.score = score;
+  }
+
+  //  stores data in array
+  scoresData = [];
+
+  // API URL
+  apiURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/gjFgjGDcJhDxNGDNGcvn/scores/';
+
+  // Show Scores
+  showScores = () => {
+    const scoresList = document.getElementById('list');
+    scoresList.innerHTML = this.scoresData.map((item) => `
+    <li>${item.user} : ${item.score}</li>`).join('');
+  };
+
+  // fetching data from API
+  fetchScores = async () => {
+    try {
+      const data = await fetch(this.apiURL);
+      const response = await data.json();
+      this.scoresData = [];
+      response.result.map((item) => this.scoresData.push(item));
+      return this.showScores();
+    } catch (error) { return error; }
+  };
```

Comment:
- [ ] [OPTIONAL] Kindly sort the scores, either alphabetically or numerically. This will ensure a better scoreboard or leaderboard viewing. check [this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) to see the know how 😉.